### PR TITLE
Functionality for listing which members will attend an event.

### PR DESCRIPTION
**Pull request by anjarfr**
Add functionality for listing members in a cohort group.

### CODE REVIEW

File: chemie/events/models.py [hunk 1]
```diff
@@ -313,6 +313,12 @@ class SocialEventRegistration(BaseRegistration):
         null=True,
         blank=True,
     )
+    registration_group_members = models.ManyToManyField(
+        User,
+        blank=True,
+        verbose_name=" ",
```
Comment (anjarfr) [inline chemie/events/models.py:319]: The verbose name is set to blank to disable the verbose name from being displayed in the template. Might be a better way to do this but idc =)

Comment (bendikstoasannes) [reply chemie/events/models.py:319]: Could be done by setting the label to an empty string in the ModelForm instead:
`fields = {"registration_group_members": ""}` as done [here](https://stackoverflow.com/a/36905089)

**Review by bendikstoasannes — APPROVED** [review]
Extraordinary work, fellow webkomers! Looking forward to påmelding tomorrow :D

File: chemie/events/models.py (hunk 1 above)
Comment (anjarfr) [reply chemie/events/models.py]: Thanks!

**Pull request merged**